sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2081,6 +2081,22 @@ object functions {
    */
   def size(column: Column): Column = Size(column.expr)
 
+  /**
+   * Returns true if the array contain the value
+   * @group collection_funcs
+   * @since 1.5.0
+   */
+  def array_contains(columnName: String, value: Any): Column =
```

Comment:
Not sure I understand what you mean. Why would it be defined for other things (such as `size` and numerous other functions), but not for `array_contains`?